package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "airbyte",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
This looks to have been generated in error and should be removed.

Reply:
Done!